src/utils/i18n/i18nTypes.ts

```diff
@@ -3,7 +3,7 @@ export type LangType = typeof supportedLangs[number]
 
 export const SUPPORTED_LANGUAGUES = [
   { label: 'English', value: 'en' },
-  { label: '中文', value: 'zh' }, // chinese
-  { label: '日本人', value: 'ja' }, // japanese
-  { label: '한국어', value: 'ko' }, // korean
+  // { label: '中文', value: 'zh' }, // chinese
```

Comment:
It's better to delete than just comment.

Reply:
Done